tests/test_modeling_common.py

```diff
@@ -2476,6 +2553,44 @@ def test_checkpoint_sharding_from_hub(self):
         for p1, p2 in zip(model.parameters(), ref_model.parameters()):
             self.assertTrue(torch.allclose(p1, p2))
 
+    def test_from_pretrained_low_cpu_mem_usage_measured(self):
+        # test that `low_cpu_mem_usage=True` uses less cpu memory than normal
+
+        mname = "bert-base-uncased"
```

Comment:
should I use another non-tiny model that we use a lot? I know I will make it @slow

Reply:
I just need something non-tiny, otherwise cpu tracker will register no difference.